src/System.Runtime.Analyzers/UnitTests/NormalizeStringsToUppercaseTests.cs

```diff
@@ -9,12 +11,332 @@ public class NormalizeStringsToUppercaseTests : DiagnosticAnalyzerTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicNormalizeStringsToUppercaseAnalyzer();
+            return new NormalizeStringsToUppercaseAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpNormalizeStringsToUppercaseAnalyzer();
+            return new NormalizeStringsToUppercaseAnalyzer();
         }
+
+        #region No Diagnostic Tests
+
+        [Fact]
+        public void NoDiagnostic_ToUpperCases()
+        {
+            VerifyCSharp(@"
+using System;
+using System.Globalization;
+
+public class UnexpectedWarningAttribute : Attribute
```

Comment:
I have just blatantly copied over the test code from the [FXCop tests](http://index/#GoodGlobalizationCS/NormalizeStringsToUppercaseTests.cs,e706e204a5bd6b1d) and tried to retain most of the original code.